ovirt-guest-agent-centos/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM centos
+MAINTAINER Vinzenz Feenstra <evilissimo@redhat.com>
+
+LABEL summary="The oVirt Guest Agent" \
+      io.k8s.description="The ovirt-guest-agent is providing information about the virtual machine and allows to restart / shutdown the machine via the oVirt Portal. This image is intended to be used with virtual machines running Centos Atomic Host." \
+      io.k8s.display-name="oVirt Guest Agent" \
+      license="ASL 2.0" \
+      architecture="x86_64"
+
+ADD logger_conf /root/logger_conf
+
+RUN dnf -y install ovirt-guest-agent-common
+RUN cat /root/logger_conf >> /etc/ovirt-guest-agent.conf && rm /root/logger_conf
+
+COPY ovirt-guest-agent-install.sh /usr/bin/install.sh
+COPY ovirt-guest-agent-uninstall.sh /usr/bin/uinstall.sh
```

Comment:
should the destination be `/usr/bin/uninstall.sh`?

I suggest putting these together in a single:

COPY ovirt-guest-agent-install.sh ovirt-guest-agent-uninstall.sh /usr/bin/